NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -666,6 +667,17 @@ class NeuroID
             }
         }
 
+        override fun setVariable(
```

Comment:
In iOS the function name is `setCustomVariable`. We probably want these to be the same.

Reply:
I thought about doing this but iOS is going to change because we want to be like trackjs and they use `setVariable`